src/PlaywrightSharp.Tests/TestConstants.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Xunit;
+using Xunit.Abstractions;
+
+namespace PlaywrightSharp.Tests
+{
+    internal static class TestConstants
+    {
+        public const string TestFixtureCollectionName = "PlaywrightSharpLoaderFixture collection";
+        public const int Port = 8081;
+        public const int HttpsPort = Port + 1;
+        public const string ServerUrl = "http://localhost:8081";
+        public const string ServerIpUrl = "http://127.0.0.1:8081";
+        public const string HttpsPrefix = "https://localhost:8082";
+        public const string AboutBlank = "about:blank";
+        public static readonly string CrossProcessHttpPrefix = "http://127.0.0.1:8081";
+        public static readonly string EmptyPage = $"{ServerUrl}/empty.html";
```

Comment:
shouldn't the static file be added too? you might forget it in the future
* same with the other properties

Reply:
What do you mean with this comment @Meir017?

Reply:
this directory - https://github.com/kblok/puppeteer-sharp/tree/master/lib/PuppeteerSharp.TestServer/wwwroot